modules/collator-selection/src/lib.rs

```diff
@@ -437,15 +449,22 @@ pub mod pallet {
 		}
 
 		fn end_session(index: SessionIndex) {
-			let mut candidates_len = 0;
 			let mut removed_len = 0;
-			<SessionPoints<T>>::drain().for_each(|(who, point)| {
-				if point == 0 {
+			let session_points = <SessionPoints<T>>::drain().collect::<Vec<_>>();
+			let candidates_len: u32 = session_points.len() as u32;
+			for (who, point) in session_points {
+				let blocks_per_session: u32 = TryInto::try_into(T::BlocksPerSession::get()).unwrap_or_default();
+				let total_session_point: u32 = blocks_per_session.checked_mul(POINT_PER_BLOCK).unwrap_or_default();
```

Comment:
Should sum all the points. Later we may have some other conditions to add more or less points. e.g. include some unsigned tx via offchain worker can give more point or failed to include it may result less reward points.